pyomo/neos/plugins/kestrel_plugin.py

```diff
@@ -218,7 +224,6 @@ def _perform_wait_any(self):
 
                 solver_results._smap_id = smap_id
                 self.results[ah.id] = solver_results
-                opt.deactivate()
```

Comment:
What did this line do and why is it not required anymore?

Reply:
Solvers used to be plugins, and references to plugins used to be kept by the Factory.  Calling `deactivate()` removed the reference from the Factory to the plpugin, thereby allowing the plugin (in this case a solver instance) to be garbage collected.  Without it, Pyomo was full of memory leaks.  It is no longer needed because Solvers are no longer PyUtilib plugins.